### PR TITLE
Minimum Sum Partition in Java

### PR DESCRIPTION
**Pull request by ritvij14**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes
(Write your answer here.)

### Description
This is the code for Minimum Sum partition in Java, which mainly follows the Recursive approach.
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
Issue #805 
(Write your answer here.)

### CODE REVIEW

**Comment by ritvij14** [comment]
@iamrajiv @Kajol-Kumari @plazzy99 @siddharth25pandey please review this.

**Review by iamrajiv — APPROVED** [review]

File: Java/other/MinimumSumPartition.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+public class MinimumSumPartition {
+
+    // Function to find the minimum sum
+    public static int findMinSum(int[] arr, int i, int sumCalculated, int total)
+    {
+        // check for last element, meaning we have sum of one subset as sumCalculated and the other as
+        // total - sumCalculated, so we return their absolute difference
+        if (i == 0) {
+            return Math.abs((total-sumCalculated) - sumCalculated);
```
Comment (vatsalkesarwani12) [inline Java/other/MinimumSumPartition.java:11]: write total - 2*sumCalculated

**Review by vatsalkesarwani12 — APPROVED** [review]

File: Java/other/MinimumSumPartition.java (hunk 1 above)
Comment (ritvij14) [reply Java/other/MinimumSumPartition.java:11]: @plazzy99 done.

**Comment by ritvij14** [comment]
@plazzy99 it is failing Prettier tests, but in the details, I cannot see where is it finding the error exactly. What should I do?

**Comment by iamrajiv** [comment]
@ritvij14 Do not worry about Prettier Action. Prettier has to be refactored.

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1:

**Comment by ritvij14** [comment]
@iamrajiv 3 approvals are done, please merge this.

**Pull request merged**